### ISSUE TITLE
Bug in Vortex v1.4.8 - Specifically Game Pass version

### ISSUE THREAD

**Issue by ChemGuy1611** [AlanParadis/Stalker2SimpleModMerger#8]
Hey, So v1.4.8 fixed the Path error when I used it on the Steam version, but for the Game Pass version, it is still coming up. 

```
Select folder containing Stalker2.exe
Game path file found
Game path loaded: F:\XboxGames\S.T.A.L.K.E.R. 2\Content
ERROR: Cannot bind argument to parameter 'Path' because it is null.
```

**Comment by AlanParadis** [comment]
I packed a  new version of 1.4.8, does it reproduce ?

**Comment by AlanParadis** [comment]
Is it fixed in 1.4.9 ?

**Comment by ChemGuy1611** [comment]
Yes, the error is gone now. Thank you!
